service/collector/ec2_instances.go

```diff
@@ -34,6 +34,9 @@ const (
 	// labelPrivateDNS will contain the private dns name
 	labelPrivateDNS = "private_dns"
 
+	// labelLifecycle shows the distinction betwen spot and on-demand.
+	labelLifecycle = "lifecycle"
```

Comment:
does it make sense to rename it to labelInstanceLifecycle to keep it consistent like the other Instance labels?

Reply:
Yes, indeed. Didn't even notice.